cli/dcoscli/node/main.py

```diff
@@ -634,6 +634,31 @@ def _get_unit_type(unit_name):
     return '{}.service'.format(unit_name)
 
 
+def _build_leader_url(component):
+    leader_prefixes_map = {
+        'dcos-marathon.service': 'marathon'
+    }
+
+    leader_url = '/leader/'
+    default_mesos_url = 'mesos'
+    postfix_log_url = '/logs/v1/'
```

Comment:
to avoid accidentally missing a / somewhere, can we make these all consistent in where the backslashes are? Maybe call `leader_url` `leader_base` and have the `leader_url` always be built up by the components here.